packages/react-examples/src/react/Dialog/Dialog.TopOffsetFixed.Example.tsx

```diff
@@ -35,10 +35,20 @@ export const DialogTopOffsetFixedExample: React.FunctionComponent = () => {
     setOptionSelected(option.key);
   };
 
+  const modalContainerCallback = React.useCallback((modalContainer: HTMLDivElement | null): void => {
+    if (modalContainer) {
```

Comment:
While this does reflect your scenario, manual DOM modifications are very much discouraged in general with React, so I don't think we should encourage people to do this by putting it in an example.

Reply:
I've removed the example.